src/schemas/json/haxelib.json

```diff
@@ -0,0 +1,93 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"description": "A haxelib project",
+	"type": "object",
+	"properties": {
+		"name": { "$ref": "#/definitions/projectName" },
+		"url": {
+			"description": "Project's website",
+			"type": "string",
+			"format": "uri"
+		},
+		"description": {
+			"description": "Short description of the project",
+			"type": "string"
+		},
+		"license": {
+			"description": "Open source license under which the project is licensed",
+			"enum": ["GPL", "LGPL", "BSD", "Public", "MIT", "Apache"]
```

Comment:
Is it limited to only these licenses? If a different license is used, then it is invalid according to this schema. To support additional licenses but keep auto-completion from the enum, create a `anyOf` for the `license` property where you add a plain `"type": "string"` to the second definition inside the `anyOf`